src/docs/testing/integration-tests/index.md

```diff
@@ -0,0 +1,285 @@
+---
+title: Integration Testing
+description: Learn how to write integration tests
+---
+
+This page describes how to use the [integration_test][] package to run
+integration tests. Tests written using this package have the following
+properties:
+ 
+- Compatibility with the `flutter drive` command, for running tests on a
+physical device or emulator.
+- The ability to be run on [Firebase Test Lab], enabling automated testing on a
```

Comment:
Please be consistent about the empty brackets (which we prefer). So "[Firebase test lab][]"'.